build.gradle

```diff
@@ -7,7 +7,7 @@ buildscript {
 
     dependencies {
         classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.8.1'
-        classpath 'com.netflix.nebula:nebula-publishing-plugin:10.0.0'
+        classpath 'com.netflix.nebula:nebula-publishing-plugin:11.0.0'
```

Comment:
apparently 11.0.0 requires gradle 5.4 - will update our internal automation so it knows this